core/src/test/java/com/sequenceiq/distrox/v1/distrox/controller/DistroxUpgradeV1ControllerTest.java

```diff
@@ -127,4 +134,11 @@ public void testUpgradeCalledWithCrn() {
         verify(upgradeService).triggerUpgrade(NameOrCrn.ofCrn(CLUSTER_NAME), WORKSPACE_ID, USER_CRN, upgradeV4Request);
     }
 
+
+    @Test
+    public void testCcmUpgrade() {
+        when(stackCcmUpgradeService.upgradeCcm(any(), any())).thenReturn(new FlowIdentifier(FlowType.FLOW, "1"));
+        ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.upgradeCcmByCrnInternal(CLUSTER_NAME, USER_CRN));
```

Comment:
Please verify the return value of `underTest.upgradeCcmByCrnInternal()`:
* Is not `null`
* Its `flowIdentifier` property matches the `FlowIdentifier` instance created above.